src/main/java/com/smartsheet/api/internal/ShareResourcesImpl.java

```diff
@@ -66,22 +67,23 @@ public ShareResourcesImpl(SmartsheetImpl smartsheet, String masterResourceType)
 	 *   SmartsheetException : if there is any other error occurred during the operation
 	 *
 	 * @param objectId the id of the object to share.
-	 * @param parameters the pagination parameters
+	 * @param pagination the pagination parameters
 	 * @param includeWorkspaceShares include workspace shares in enumeration
 	 * @return the shares (note that empty list will be returned if there is none)
 	 * @throws SmartsheetException the smartsheet exception
 	 */
 	public PagedResult<Share> listShares(long objectId, PaginationParameters pagination) throws SmartsheetException {
 		return this.listShares(objectId, pagination, false);
 	}
+
 	public PagedResult<Share> listShares(long objectId, PaginationParameters pagination, Boolean includeWorkspaceShares) throws SmartsheetException {
 		String path = getMasterResourceType() + "/" + objectId + "/shares";
 		
 		HashMap<String, Object> parameters = new HashMap<String, Object>();
 		if (pagination != null) {
 			parameters = pagination.toHashMap();
 		}
-		if (includeWorkspaceShares == true) {
+		if (includeWorkspaceShares != null && includeWorkspaceShares == true) {
```

Comment:
why is includeWorkspaceShares a Boolean to begin with?  if it were boolean there would be no problem here (also the "== true" is redundant).